pmoves/services/channel-monitor/channel_monitor/main.py

```
@@ -105,20 +131,62 @@ def _extract_urls_from_text(value: str | None) -> list[str]:
 @asynccontextmanager
 async def lifespan(app: FastAPI):
     """Manage Channel Monitor application lifespan."""
+    global _nats_client
     # Startup
     await monitor.start()
     app.state.monitor = monitor
+    if NATS_AVAILABLE and CONTENT_RAW_PUBLISH_ENABLED:
+        try:
+            _nats_client = await nats_pkg.connect(NATS_URL)
+            LOGGER.info("content.raw.v1 publisher connected to %s", NATS_URL)
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)